cupy/cuda/compiler.py

```diff
@@ -1,36 +1,34 @@
 import hashlib
 import os
 import re
-import subprocess
-import sys
 import tempfile
 
 import filelock
 import six
 
 from cupy.cuda import device
 from cupy.cuda import function
+from pynvrtc.compiler import NVRTCInterface
+from pynvrtc.compiler import Program
 
+_nvrtc_version = None
 
-_nvcc_version = None
 
+def _get_nvrtc_version():
+    global _nvrtc_version
```

Comment:
Hmm... I'm not sure. I checked the document of NVRTC, but couldn't find out whether `nvrtcVersion()` is thread safe or not. Could you give me a hint how to know its thread safety?